docs/src/examples/quick-start.md

```diff
@@ -0,0 +1,101 @@
+# Quick Start
+
+The package is hosted on maven central repository. To build from source, refer to the [Contribution Guide](../contribution/introduction.html).
+
+## Create a maven project
+
+First download [maven] and follow the [installation instructoins][install]. Then `mvn` command should be available in the `$PATH`.
```

Comment:
It's a [shortcut reference link](https://spec.commonmark.org/0.30/#shortcut-reference-link), the link defination is several lines below. I prefer this form of link because it makes the markdown text cleaner and easier to read.

https://github.com/tikv/client-java/blob/79d425022f2b25b534fdf806dd8e952c0db22f00/docs/src/examples/quick-start.md#L16-L17

Reply:
Done.